pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -290,6 +290,12 @@ protected void internalDeleteTenant(AsyncResponse asyncResponse, String tenant)
     }
 
     protected void internalDeleteTenantForcefully(AsyncResponse asyncResponse, String tenant) {
+        if (!pulsar().getConfiguration().isForceDeleteTenantAllowed()) {
+            asyncResponse.resume(
+                    new RestException(Status.FORBIDDEN, "Broker doesn't allow forced deletion of tenants"));
```

Comment:
The @ApiReponse use 403 to indicate if the request has admin permission, I think 405 is more reasonable here?

Reply:
Thanks for the tips, done.